bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -449,4 +449,86 @@
 		<category>thermostat</category>
 	</channel-type>
 
+	<channel-type id="smokeDetection">
+		<item-type>Switch</item-type>
+		<label>Smoke Detected</label>
+		<description>Smoke detection sensor state.</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="sensorFault">
+		<item-type>Switch</item-type>
+		<label>Sensor Fault Mode</label>
+		<description>Sensor activated fault mode.</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="maintenanceStatus">
+		<item-type>Switch</item-type>
+		<label>Maintenance neccessary</label>
+		<description>Sensor maintenance status.</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="saHumidityRange">
+		<item-type>Switch</item-type>
+		<label>Out of Humidity Range</label>
+		<description>Sensor analysis status: humidity out of range</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="saTemperatureRange">
+		<item-type>Switch</item-type>
+		<label>Out of Temperature Range</label>
+		<description>Sensor analysis status: temperature out of range</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="timeSinceLastMaintenance">
+		<item-type>Number</item-type>
+		<label>Weeks Since Last Maintenance</label>
+		<description>Time since last maintenance was done in weeks.</description>		
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="remainingPLT">
+		<item-type>Number</item-type>
+		<label>Remaining Product Life Time in Months</label>
+		<description>Countdown time until product end of life.</description>		
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="hygroComfortIndex">
+		<item-type>Number</item-type>
+		<label>Hygrothermal Comfort Index</label>
+		<description>Comfort index based on temperature and humidity</description>
+		<state>
+			<options>
```

Comment:
This is only supposed to be used for String item-types, so could you make it a string and map the numeric values to some meaningful values, such as GOOD/MEDIUM etc (use an enum)

> Some channels might have only a limited and countable set of states. These states can be specified as options. A String item must be used as item type.
[[source]](https://www.openhab.org/docs/developer/bindings/thing-xml.html#state-description)

Reply:
Changed to StringType, however I did not used an enum, a simple string array is enough. Channel is readonly so int => string conversion is all I need. Opposite way is not needed.
BTW: Number item worked without any problems, too.

Reply:
I meant to have to value's themselves more descriptive, now the logical mapping to String is only available in the paper UI other UI's only have a number which is not at all self-explanatory.